jest.config.js

```diff
@@ -0,0 +1,8 @@
+// Copied from https://jestjs.io/docs/code-transformation#examples
+module.exports = {
+  ...require('@wordpress/scripts/config/jest-unit.config'),
+  transform: {
+    '^.+\\.[jt]sx?$': '<rootDir>/node_modules/@wordpress/scripts/config/babel-transform',
+    '\\.(jpg|jpeg|png|svg)$': '<rootDir>/fileTransformer.js',
```

Comment:
Babel knows what to do when it finds non-JS files, like `.svg` or `.png`.

But it looks like Jest doesn't.

So this adds a transform copied from the [Jest docs](https://jestjs.io/docs/code-transformation#examples)